changelogs/fragments/2373-svr4pkg-fix-typeerror.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - svr4pkg - convert "fullauto" string to a bytes-like object to avoid TypeError
```

Comment:
```suggestion
  - svr4pkg - convert "fullauto" string to a bytes-like object to avoid TypeError (https://github.com/ansible-collections/community.general/issues/2373).
```